addons/crm/models/crm_lead.py

```diff
@@ -230,6 +264,34 @@ def _compute_email_state(self):
                         break
             lead.email_state = email_state
 
+    @api.depends('probability', 'automated_probability')
+    def _compute_is_automated_probability(self):
+        """ If probability and automated_probability are equal probability computation
+        is considered as automatic, aka probability is sync with automated_probability """
+        for lead in self:
+            # creation mode: consider it as being not automated
+            if not lead.id and not lead._origin.id:
+                lead.is_automated_probability = False
+            else:
+                lead.is_automated_probability = tools.float_compare(lead.probability, lead.automated_probability, 2) == 0
+
+    @api.depends(lambda self: ['tag_ids', 'stage_id', 'team_id'] + self._pls_get_safe_fields())
+    def _compute_probabilities(self):
+        for lead in self:
+            was_automated = False
+            lead_probabilities = lead._pls_get_naive_bayes_probabilities()
+            if lead.id in lead_probabilities:
+                if tools.float_compare(lead.probability, lead.automated_probability, 2) == 0 and lead.active:
+                    was_automated = True
+                lead.automated_probability = lead_probabilities[lead.id]
+                if was_automated:
+                    lead.probability = lead.automated_probability
```

Comment:
why not re-using is_automated_probability ?

Reply:
Indeed.